sharding/data_modeling.rst

```diff
@@ -26,10 +26,77 @@ Distributing by Tenant ID
 
 The multi-tenant architecture uses a form of hierarchical database modeling to distribute queries across nodes in the distributed cluster. The top of the data hierarchy is known as the *tenant id*, and needs to be stored in a column on each table. Citus inspects queres to see which tenant id they involve and routes the query to a single physical node for processing, specifically the node which holds the data shard associated with the tenant id. Running a query with all relevant data placed on the same node is called *co-location*.
 
+Two tables co-located for queries joining on account id:
+
+.. figure:: ../images/mt-colocation.png
+   :alt: co-located tables in multi-tenant architecture
+
 The first step is identifying what constitutes a tenant in your app. Common instances include company, account, organization, or customer. The column name will be something like :code:`company_id` or :code:`customer_id`. Examine each of your queries and ask yourself: would it work if it had additional WHERE clauses to restrict all tables involved to rows with the same tenant id? Queries in the multi-tenant model are usually scoped to a tenant, for instance queries on sales or inventory would be scoped within a certain store.
 
 If you're migrating an existing database to the Citus multi-tenant architecture then some of your tables may lack a column for the application-specific tenant id. You will need to add one and fill it with the correct values. This will denormalize your tables slightly. For more details and a concrete example of backfilling the tenant id, see our guide to :ref:`Multi-Tenant Migration <transitioning_mt>`.
 
+Typical Multi-Tenant Scenario
```

Comment:
Since we only have one example, could we try the following rewording?

"Typical Multi-Tenant Scenario

Advertising Analytics

In this example (partially explained by the diagram above), the Account table is chosen as the unit of tenancy. Imagine a web advertising analytics platform-as-a-service, where each customer has an account and tracks advertising clicks for various campaigns." -->

"Typical Multi-Tenant Schema

Most SaaS applications already have the notion of tenancy built into their data model. In the following, we will look at an example schema from the online advertising space. In this example, a web advertising platform has tenants that it refers to as accounts. Each account holds and tracks advertising clicks across various campaigns."